buildkite/bazelci.py

```diff
@@ -861,20 +861,36 @@ def rbe_flags(accept_cached):
         "--java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8",
         "--crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/latest:crosstool_top_default",
         "--action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1",
-        # Platform flags:
-        # The toolchain container used for execution is defined in the target indicated
-        # by "extra_execution_platforms", "host_platform" and "platforms".
-        # If you are using your own toolchain container, you need to create a platform
-        # target with "constraint_values" that allow for the toolchain specified with
-        # "extra_toolchains" to be selected (given constraints defined in
-        # "exec_compatible_with").
-        # More about platforms: https://docs.bazel.build/versions/master/platforms.html
         "--extra_toolchains=@bazel_toolchains//configs/ubuntu16_04_clang/latest:toolchain_default",
```

Comment:
Should these be deleted?

Reply:
There are moved outside.

Reply:
Oh, my mistake, they should be deleted.

Reply:
Done in https://github.com/bazelbuild/continuous-integration/commit/298e6155a80913d0c81993228ea38b4151ee1bd0